config/300-pipelinerun.yaml

```diff
@@ -26,20 +26,16 @@ spec:
     - tekton-pipelines
   scope: Namespaced
   additionalPrinterColumns:
-    - name: Type
+    - JSONPath: .status.conditions[?(@.type=="Succeeded")].status
+      description: The status of the pipelinrun
+      name: Status
       type: string
-      description: The cron spec defining the interval a CronJob is run
-      JSONPath: .status.conditions[0].type
-    - name: Status
-      type: string
-      description: The number of jobs launched by the CronJob
-      JSONPath: .status.conditions[0].status
-    - name: StartTime
+    - JSONPath: .status.startTime
+      name: StartTime
       type: date
-      JSONPath: .status.startTime
-    - name: CompletionTime
+    - JSONPath: .status.conditions[?(@.reason!="Running")].CompletionTime
```

Comment:
`.status.completionTime` :pray:

Reply:
FYI: c'est fait ;) (and thanks for the review)